webapp/src/components/lhs/item.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {useState} from 'react';
+import {ListItem, ListItemText, IconButton, Menu, MenuItem, ListItemIcon} from '@mui/material';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import {CheckCircleOutline, StarBorderOutlined} from '@mui/icons-material';
+
+
+interface ItemProps {
+    id: string;
+    display_name: string;
+    areaLabel: string;
+    link: string;
+    isCollapsed: boolean;
+}
+
+const Item = (props: ItemProps) => {
+    const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+    const [isHovered, setIsHovered] = useState(false);
+    const open = Boolean(anchorEl);
+
+    const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    if (props.isCollapsed) {
```

Comment:
Why should the item know if the group is collapsed or not?

Reply:
If group is collapsed all items are collapsed too